sql/files/defaultdata/adb/run

```diff
@@ -10,9 +10,3 @@ MAINSOURCE="$1"
 # -static:         Static link with all libraries
 # -bargs -static:  Static link with all libraries
 gnatmake -static -o "$DEST" "$@" -bargs -static
-exitcode=$?
```

Comment:
Why did you remove these?

Reply:
See the commit message, I assumed this was from a time where we didn't have the chroot yet so had to do cleanup.

Reply:
@ubergeek42 added this in 2013 in 460c79ab16a0695dca8a120227d319af09cdf0b0

Keith, do you remember why you cleaned up the files?

Reply:
Also, don't remove the `exit $exitcode` as it detects and returns whether compilation was successful.

Reply:
Aah, ok I'll just revert that whole commit.